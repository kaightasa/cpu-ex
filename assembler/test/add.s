calc:
	addi r2, r1, 16
	add r4, r3, r2
