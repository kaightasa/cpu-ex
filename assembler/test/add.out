8410010
c831000
